document/image-format-negotiation.md

```
@@ -0,0 +1,10 @@
+- Image format negotiation:
+
+    * is the process to ensure that the camera driver and application agree on the same:
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: dash; Actual: asterisk

(MD004, ul-style)